library/src/main/java/com/bartoszlipinski/viewpropertyobjectanimator/PercentChangeListener.java

```diff
@@ -17,7 +17,7 @@
 
 import android.animation.ValueAnimator;
 import android.animation.ValueAnimator.AnimatorUpdateListener;
-import android.support.percent.PercentLayoutHelper;
+import androidx.percentlayout.widget.PercentLayoutHelper;
 import android.view.View;
```

Comment:
`Support` library is mentioned in the error messages in this class. Please modify them as well.

Reply:
Do you mean the deprecation of PercentLayout?

Reply:
Those lines:
https://github.com/blipinsk/ViewPropertyObjectAnimator/blob/f0409606ce8fece46f6d90ec499478505e335699/library/src/main/java/com/bartoszlipinski/viewpropertyobjectanimator/PercentChangeListener.java#L46-L52

mention `part of the Percent Support Library`. 

This needs to be modified because it's no longer part of the **Support** library.